File: quote/Projects/SampleProperties/OtherProperiesFactory.cs
using System;

using Model.Template;
using Host;

namespace SampleProperties
{
    [Register(Key = typeof(Model.Template.Ext.IOtherPropertiesFactory))]
    public class OtherProperiesFactory : Model.Template.Ext.IOtherPropertiesFactory
    {
        public OtherProperiesFactory()
        {
        }

        public Model.Common.SaveableProperties CreateOtherProperties(
            Header header, int id)
        {
            return new SampleOtherProperties(header);
        }

    }
}

File: quote/Projects/DekalbProperties/OtherProperiesFactory.vb
Imports Host
Imports Model.Template.Ext

<Register(Key:=GetType(Model.Template.Ext.IOtherPropertiesFactory))>
Public Class OtherProperiesFactory
    Implements IOtherPropertiesFactory

    Public Function CreateOtherProperties(header As Model.Template.Header, _
                                          id As Integer) As  _
                                      Model.Common.SavableProperties _
                                      Implements IOtherPropertiesFactory.CreateOtherProperties
        Return New DekalbOtherProperties(header)
    End Function

End Class
